### ISSUE TITLE
FeelParser Cannot Parse Multiple Intervals Combined With Disjunction

### ISSUE THREAD

**Issue by evangaito** [camunda/feel-scala#349]
**Describe the bug**
A parse error occurs when multiple intervals are combined using disjunction in a unary test.

**To Reproduce**
1. See below output from ammonite. A single interval correctly evaluated to true/false but multiple intervals separated either by "," or "or" fail with a parse error.
```
@ unaryTests("[160..162]", 160)
> true


@ unaryTests("[160..162]", 166)
> false


@ unaryTests("[166..168]", 166)
> true


@ unaryTests("[160..162], [166..168]", 166)
> Failure(failed to parse expression '[160..162], [166..168]': Expected end-of-input:1:11, found ", [166..16")


@ unaryTests("[160..162] or [166..168]", 166)
> Failure(failed to parse expression '[160..162] or [166..168]': Expected end-of-input:1:12, found "or [166..1")
```

**Expected behavior**
Each interval is treated as an individual unary test expression that can then be combined with disjunction.

**Environment**
* FEEL engine version: 1.13.3
* Affects: 
  * Camunda BPM: 7.13+ Unsure if CAM issue exists
  * Zeebe broker: Unknown

**Comment by saig0** [comment]
@evangaito thank you for creating the issue :+1: 
You could create also a Jira issue for Camunda Platform but you don't need to. We can track the status in this issue.

**Comment by saig0** [comment]
The bug is already fixed in version `1.4.0`. I added a new test case for this unary-tests expression with #376.
